jest.config.js

```diff
@@ -11,6 +11,10 @@ module.exports = {
   testEnvironment: 'jsdom',
   setupFilesAfterEnv: ['<rootDir>/jest-setup.ts'],
   moduleNameMapper: {
+    // ref: https://react-hooks-testing-library.com/installation#being-specific
+    // This mapping will ensure the correct renderHook is used in the test environment
+    // as the import within `utils/testing` is for the specific dom version.
+    '@testing-library/react-hooks/dom': '@testing-library/react-hooks',
```

Comment:
Without this, the wrong import is used in tests. In a build, if just using the import `'@testing-library/react-hooks'` - the wrong import is used. 

So.. use  the import `'@testing-library/react-hooks/dom';` and then remap when in a test env

Technically this is because `utils/testing.ts` is being bundled with the app in the dom, when it shouldn't be (should be tree shaken and only used in tests).